LiteCore/Query/SQLiteFleeceEach.cc

```diff
@@ -176,7 +177,7 @@ namespace litecore {
 
 #pragma mark - INSTANCE METHODS:
 
-        FleeceCursor(FleeceVTab* vtab) : _vtab(vtab) {}
+        explicit FleeceCursor(FleeceVTab* vtab) : sqlite3_vtab_cursor{vtab}, _vtab(vtab) {}
```

Comment:
What is the righthand side of this change doing?

Reply:
Initializes the base class `sqlite3_vtab_cursor`. 
There is another constructor for the object:
`explicit FleeceCursor(FleeceVTab* vtab) : sqlite3_vtab_cursor{vtab}, _vtab(vtab) {}`
Which initializes both the base class, and `_vtab`.
Interestingly, `_vtab` has the same data type as the base class, so it's possible we only need one of them. Going to investigate that now.

Reply:
Hmm yeah, removing the inheritance from `sqlite3_vtab_cursor`, although it causes no compile errors, causes fatal errors during QueryTest. So I guess some sqlite3 functions that takes pointers etc, rely on the passed object inheriting from sqlite3_vtab_cursor.